### PR TITLE
More documentation on setting up a local environment

### PR DESCRIPTION
**Pull request by ericandrewlewis**

### CODE REVIEW

**Comment by joseph-onsip** [comment]
:+1:

**Comment by abitrolly** [comment]
This needs to be rebased to solve conflicts.

File: README.md [hunk 1]
```diff
@@ -53,7 +54,12 @@ To setup a working environment that lets you run the web interface locally, you'
 
 ### Processing a man page
 
-Use the manager to parse and save a gzipped man page in raw format:
+Manpages are processed from gzipped source into a useful format in the MongoDB before
+they can be served by the explainshell front-end.
+
+A few manpage gzipped sources are included in [manpages/1/](manpages/1/) and [manpages/8/](manpages/8/).
+
+Use the manager process one:
```
Comment (abitrolly) [inline README.md:62]:
```suggestion
Use `explainshell/manager.py` script to load pages into MongoDB:
```

**Comment by abitrolly** [comment]
@ericandrewlewis will you have the time to rebase this to cleanup PRs a bit?

**Comment by ericandrewlewis** [comment]
No I'm not working on this, feel free to create a new PR

**Pull request closed without merging**